javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -250,6 +251,27 @@ public void shouldNothingReturnNotNull() {
         assertThat(Nothing()).isNotNull();
     }
 
+    @Test
+    public void shouldTryReturnNotNull() {
+        Try<Object> _try = Try(() -> 1);
```

Comment:
Minor suggestions:
* consider making local variables `final` (@danieldietrich likes it that way)
* you could name this `t` instead of starting with a weird underscore, since the scope is very small